src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,50 @@
+# Navigate Between Windows and Applications
+
+Pop!\_OS uses the GNOME desktop environment, which includes keyboard shortcuts for quickly switching between system and application windows.
+
+---
+
+## Navigate Between Windows
```

Comment:
I think the sequence of this section, the following keyboard navigation section, and the Workspaces page could potentially benefit from more consideration. At the least, since this section talks about using the mouse to select a window, I would expect it to include that clicking Workspaces in the upper-left or in the dock can also open this menu as an alternative to the keyboard shortcut.

This page/section also omits perhaps the most obvious method of switching between open windows using the dock. Another thing that's missing is the ability to switch between open windows with the Launcher (again, might be useful to mention it here but link to a page just going over the Launcher's many features, just an idea.)

Reply:
I want Workspaces to have its own title section because it's a significant feature. It's hard to segment Workspaces from other window-switching methods because SUPER + D is one of the most useful ways to "find your hidden window". 

Thanks for reminding me of the obvious methods that I overlooked. Here are the changes I'll make: 

- "Switch Between Applications & Windows" will become a section instead of a subsection
- Remove mention of Workspaces from the above section
- Add the two methods you mentioned to the above section
- Move "Workspaces" above "Tile & Stack Windows" so it comes after "Switch Between Applications & Windows"
- Keyboard shortcuts will be combined and listed at the end of each section